certif/app/styles/globals/colors.scss

```diff
@@ -19,3 +19,4 @@ $tundora: #444444;
 // Gradients
 $blue-purple-gradient: linear-gradient(135.25deg, $dodger-blue 0%, $heliotrope 100%);
 $denim-gradient: linear-gradient(0deg, $denim 0%, $astronaut 100%);
+$grey-gradient: linear-gradient(0deg, #585F75 0%, #414657 100%);
```

Comment:
Nommer les couleurs ([src](http://chir.ag/projects/name-that-color/#414657)) : 
- #585F75 → "$comet"
- #414657 → "$gun-powder"